exercises/mark-griffith/short-response.md

```diff
@@ -9,12 +9,12 @@
 
 2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments?
 3. 
-     Invoking a JavaScript function with missing arguement would run but return either undefined since thats the value that is placed in its stance.
-     If extra arguements are passed JavaScript runs with the first one that is passed.
+     Invoking a JavaScript function with missing arguement would run but pass either undefined as the value since JavaScript will alway try to continue to run.
+     If extra arguements are passed JavaScript will ignore them.
 
 3. Define _hoisting_. What impact does hoisting have on the way that we structure our JavaScript programs?
     
-    Hoisting is the process of JavaScript lifting up funtions as they are invoked. 
+    Hoisting is the process of JavaScript lifting up funtions as they are called. 
```

Comment:
This answer still needs some work but will approve this submission as we will talk more about hoisting in class.